### PR TITLE
#158422386 User should be able to see all the available ride offers #158422553 User should be able to see the details of  a ride offer and respond to it

### PR DESCRIPTION
**Pull request by LarryKarani**
**What does this PR do?**
This pull request creates a ride offer view page and a details view page and links the two together

**Description of Task to be completed?**
The ride offer view page should contain all the ride offers and a link to to its details 
The ride offer details page should contain all the details about the ride offer
**How the app should be tested**
Clone the repo to your local machine
Navigate to Ride My Way then to ui
open the rideoffers.html  file using any suitable browser (preferably chrome)

**Any background context you want to provide?**
The User Interface has no functionality.
The Details page is a mock up and doesnt depict the real details of the ride offer 
Developed using HTML CSS

**What are the relevant pivotal tracker stories?**
https://www.pivotaltracker.com/n/projects/2179558
[#158422386] [#158422553]

**all ride offers page full-screen**
offer 1

**full screen ride offer details page**
of4

**small screen ride offer details page**
of3

### CODE REVIEW

File: UI/css/style.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+.ride_of, .details{
+    
+    max-width:700px;
+    background-color: silver;
+    padding: 40px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    letter-spacing: 1px; 
+    margin: auto;
+    /* text-align: center; */
```
Comment (justmesam) [inline UI/css/style.css]: avoid pushing commented code..

File: UI/css/style.css [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    
+
+}
+#img{
+    text-align: center;  
+}
+
+/* #hidden{ */
+    /* display: none; */
+/* } */
```
Comment (justmesam) [inline UI/css/style.css]: Avoid pushing commented code

**Review by justmesam — COMMENTED** [review]
work on the comments I have added

File: UI/css/style.css (hunk 2 above)
Comment (LarryKarani) [reply UI/css/style.css]: Thanks i will remove the comments

**Pull request closed without merging**